src/styles/theme/defaultTheme.ts

```diff
@@ -499,7 +499,7 @@ export const defaultTheme = createTheme({
       styleOverrides: {
         paperAnchorTop: ({ theme }) => ({
           borderRadius: '0 0 12px 12px',
-          padding: `${theme.spacing(4)} 5%`,
+          padding: theme.spacing(4, '5%'),
```

Comment:
it doesn't have negative numbers, but has percentages? :O

Reply:
yeah it looks like it's not supported